modules/cp4ba/main.tf

```diff
@@ -11,35 +9,46 @@ locals {
   roles_file_content                    = file(local.roles_file)
   role_binding_file                     = "${path.module}/files/role_binding.yaml"
   role_binding_content                  = file(local.role_binding_file)
-//  cp4ba_subscription_file               = "${path.module}/files/cp4ba_subscription.yaml"
-//  cp4ba_subscription_file_content       = file(local.cp4ba_subscription_file)
   cp4ba_subscription_file_content       = templatefile("${path.module}/templates/cp4ba_subscription.yaml.tmpl", {
-    namespace = var.cp4ba_project_name
+    cp4ba_project_name = var.cp4ba_project_name
   })
-  cp4ba_deployment_credentials_file     = "${path.module}/templates/cp4ba_deployment_credentials.yaml.tmpl"
-  cp4ba_deployment_credentials_file_content = file(local.cp4ba_deployment_credentials_file)
-  cp4ba_deployment_file_content         = templatefile("${path.module}/templates/cp4ba_deployment.yaml.tmpl", {
+  operator_shared_pv_file_content   = templatefile("${path.module}/templates/operator_shared_pv.yaml.tmpl", {
+    cp4ba_project_name = var.cp4ba_project_name
+  })
+  shared_log_pv_file_content        = templatefile("${path.module}/templates/cp4a_shared_log_pv.yaml.tmpl", {
+    cp4ba_project_name = var.cp4ba_project_name
+  })
+  operator_shared_pvc_file_content  = templatefile("${path.module}/templates/operator_shared_pvc.yaml.tmpl", {
+    cp4ba_project_name = var.cp4ba_project_name
+  })
+  shared_log_pvc_file_content       = templatefile("${path.module}/templates/cp4a_shared_log_pvc.yaml.tmpl", {
+    cp4ba_project_name = var.cp4ba_project_name
+  })
+  cp4ba_deployment_credentials_file_content = "${path.module}/templates/cp4ba_deployment_credentials.yaml.tmpl"
+  cp4ba_deployment_file_content             = templatefile("${path.module}/templates/cp4ba_deployment.yaml.tmpl", {
     ldap_host_ip     = var.ldap_host_ip,
-    db2_admin        = var.db2_admin,
-    db2_host_name    = var.db2_host_name,
+    db2_host_address    = var.db2_host_address,
     db2_host_port    = var.db2_host_port,
     ingress_subdomain = var.ingress_subdomain
   })
-
-  secrets_content = templatefile("${path.module}/templates/secrets.yaml.tmpl", {
-    ldap_admin       = var.ldap_admin,
-    ldap_password    = var.ldap_password,
-    db2_admin        = var.db2_admin,
+  secrets_content    = templatefile("${path.module}/templates/secrets.yaml.tmpl", {
+    ldap_admin       = var.ldap_admin_name,
+    ldap_password    = var.ldap_admin_password,
+    db2_admin        = var.db2_admin_username,
     db2_user         = var.db2_user,
-    db2_password     = var.db2_password
+    db2_password     = var.db2_admin_user_password
   })
 }
 
 resource "null_resource" "installing_cp4ba" {
-  count = var.enable ? 1 : 0
+//  enable_cp4ba = var.enable_cp4ba
```

Comment:
Is this in or out?  You are still referring to it for the `get_endpoints` data resource.  This `enable` flag isn't really needed for these cloud pak modules anymore, so ok to remove.